eng/Baseline.Designer.props

```diff
@@ -966,7 +966,7 @@
     <BaselinePackageReference Include="Microsoft.AspNetCore.Hosting" Version="[2.2.0, )" />
     <BaselinePackageReference Include="Microsoft.Net.Http.Headers" Version="[2.2.0, )" />
     <BaselinePackageReference Include="Microsoft.Win32.Registry" Version="[4.5.0, )" />
-    <BaselinePackageReference Include="System.Security.Principal.Windows" Version="[4.5.0, )" />
+    <BaselinePackageReference Include="System.Security.Principal.Windows" Version="[4.5.1, )" />
```

Comment:
This is not correct and I'll revert it soon. Microsoft.AspNetCore.Server.HttpSys v.2.2.6 still depends on System.Security.Principal.Windows v4.5.0.

@wtgodbe and @mmitche how did we get into this state -- depending on different versions of this package in `release/2.1` and `release/2.2`? Is there an adjustment we should make in the next 2.2.x patch?

#### Details

We got a new System.Security.Principal.Windows version in 2.1.6 ProdCon builds. In 6effb13bbdd0, Nate changed its version in our build/dependencies.props file too. But, nothing similar has happened in 2.2.x builds and Microsoft.AspNetCore.Server.HttpSys v2.1.12 is the only ASP.NET Core package we've released depending on System.Security.Principal.Windows v4.5.1.

Reply:
> We got a new System.Security.Principal.Windows version in 2.1.6 ProdCon builds. In 6effb13, Nate changed its version in our build/dependencies.props file too. But, nothing similar has happened in 2.2.x builds and Microsoft.AspNetCore.Server.HttpSys v2.1.12 is the only ASP.NET Core package we've released depending on System.Security.Principal.Windows v4.5.1.

Maybe a merge between 2.1 and 2.2 was resolved incorrectly?

Reply:
We couldn't merge the build/dependencies.props change into 'release/2.2' because ProdCon was still flowing System.Security.Principal.Windows v4.5.0. Must be flowing that version or our builds would fail. That said, is the System.Security.Principal.Windows version correct (updated) elsewhere in 2.2?

Reply:
We discussed this offline and #12129 and #12132 were the result